--- pydm/data_plugins/epics_plugin.py	2018-04-04 08:18:25.582762765 -0700
+++ pydm/data_plugins/epics_plugin.py	2018-04-04 08:18:51.916528786 -0700
@@ -12,9 +12,9 @@
     EPICSPlugin = PSPPlugin
 else:
     try:
-        from pydm.data_plugins.epics_plugins.psp_plugin_component import PSPPlugin
-        EPICSPlugin = PSPPlugin
-    except ImportError:
         from pydm.data_plugins.epics_plugins.pyepics_plugin_component import PyEPICSPlugin
         EPICSPlugin = PyEPICSPlugin
+    except ImportError:
+        from pydm.data_plugins.epics_plugins.psp_plugin_component import PSPPlugin
+        EPICSPlugin = PSPPlugin
 EPICSPlugin.protocol = "ca"
