--- EPICS-CPP-4.6.0/pvaPy/tools/autoconf/m4/ax_pva_py.m4	2017-03-02 11:16:29.247099700 -0800
+++ EPICS-CPP-4.6.0/pvaPy/tools/autoconf/m4/ax_pva_py.m4	2017-03-02 11:03:09.572208000 -0800
@@ -148,6 +148,6 @@
     echo "PVA_API_VERSION = $PVA_API_VERSION" >> $config_site_local
     echo "PVA_RPC_API_VERSION = $PVA_RPC_API_VERSION" >> $config_site_local
     echo "HAVE_BOOST_NUM_PY = $HAVE_BOOST_NUM_PY" >> $config_site_local
-    echo "PYTHON_VERSION := \$(shell python -c 'import sys; print(sys.version[[:3]])')" >> $config_site_local
+    echo "PYTHON_VERSION := \$(shell python -c 'import sys; print sys.version[[:3]]')" >> $config_site_local
     AC_MSG_NOTICE([created $config_site_local file])
 ])
