--- EPICS-CPP-4.6.0/pvaPy/tools/autoconf/configure.ac	2016-09-18 10:22:36.000000000 -0700
+++ EPICS-CPP-4.6.0/pvaPy/tools/autoconf/configure.ac	2017-03-02 11:14:22.656492100 -0800
@@ -2,7 +2,7 @@
 AM_INIT_AUTOMAKE([-Wall -Werror foreign])
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_MACRO_DIR([m4])
-AX_PYTHON_DEVEL([>=2.6])
+AX_PYTHON_DEVEL([>='2.6'])
 AX_BOOST_BASE([1.40], [], [AC_MSG_ERROR(required Boost library version >= 1.40.)])
 AX_BOOST_PYTHON
 AX_BOOST_NUM_PY
